### PR TITLE
FEATURE: Update ligand data from GTP 

### PR DESCRIPTION
**Pull request by AlibekMamyrbekov**
Update ligands data using GTP data:
1) update ligands
2) update ligand assays
3) update endogenous ligands

### CODE REVIEW

File: tools/management/commands/get_gtp_ligands.py [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+            try:
+                temp = dict()
+                temp['entry'] = entry['ligandId']
+                temp['name'] = entry['name']
+                temp['type'] = entry['type']
+                self.save_ligand_copy(temp)
+            except:
+                pass
+
+    def get_gpcrs(self):
```
[the remark this answers is not in the document]
Comment (AJK-dev) [reply tools/management/commands/get_gtp_ligands.py:135]: Example are the orphans

**Review by AJK-dev — CHANGES_REQUESTED** [review]
@AlibekMamyrbekov: There are some potential issues, could you take a look at the comments and then let's book a meeting

File: ligand/models.py [hunk 2]
```diff
@@ -313,6 +387,11 @@ def __str__(self):
     class Meta():
         db_table = 'ligand_role'
 
+class LigandReceptorStatistics(models.Model):
```
[the remark this answers is not in the document]
Comment (AJK-dev) [reply ligand/models.py:390]: Discussed, need further specifications for the selectivity index. Then update this table

File: tools/management/commands/get_gtp_ligands.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AJK-dev) [reply tools/management/commands/get_gtp_ligands.py:135]: Add a check for the orphan and other level:
`{
  "familyId" : 115,
  "name" : "Orphan and other 7TM receptors",
  "targetIds" : [ ],
  "parentFamilyIds" : [ 694 ],
  "subFamilyIds" : [ 16, 706, 18, 975, 116, 117, 113 ]
}`

**Review by AlibekMamyrbekov — COMMENTED** [review]
DONE

File: tools/management/commands/get_gtp_ligands.py [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+            ligand['name'], ligand['entry'], ligand['type'])
+        if l != None:
+            self.ligand_cache[ligand['entry']] = l
+
+    def get_ligands(self):
+        ligands = list()
+        response = requests.get(
+            "https://www.guidetopharmacology.org/services/ligands")
+        print('total ligands:', len(response.json()))
+        for entry in response.json():
```
[the remark this answers is not in the document]
Comment (AlibekMamyrbekov) [reply tools/management/commands/get_gtp_ligands.py]: DONE

**Comment by AlibekMamyrbekov** [comment]
major refactoring

**Pull request closed without merging**